src/openrct2/peep/Staff.cpp

```diff
@@ -386,8 +378,12 @@ void staff_reset_stats()
     }
 }
 
-bool staff_is_patrol_area_set(int32_t staffIndex, int32_t x, int32_t y)
+static bool staff_is_patrol_area_set(int32_t staffIndex, int32_t x, int32_t y)
 {
+    // Patrol quads are stored in a bit map (8 patrol quads per byte)
+    // Each patrol quad is 4x4
+    // Therefore there are in total 64 x 64 patrol quads in the 256 x 256 map
+
```

Comment:
May as well add a comment that staff types have a common patrol area in the array as well after the `200 (STAFF_MAX_COUNT)`.